ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/classload/ClassLoadInvoker.java

```diff
@@ -246,8 +248,10 @@ public void reset() {
         Collection<GlobalVariableSymbol> globalVariableSymbols = globalVariableSymbols(project, compilation);
         Map<QuotedIdentifier, GlobalVariable> allNewVariables = new HashMap<>();
         for (VariableDeclarationSnippet snippet : variableDeclarations.keySet()) {
+            boolean isVarAssigned = snippet.getRootNode().typedBindingPattern()
+                    .typeDescriptor().kind().equals(SyntaxKind.VAR_TYPE_DESC);
```

Comment:
No need to use `equals()` with enum. Can just use `==`. Also, do we need to be exposing this root node out of the var decl snippet? IINM, this is the only place it's used in right? Should we consider encapsulating this logic within the var decl snippet itself? 🤔

Reply:
Updated in d3e7d2d557259712428f4f178b8d6bfd67325435